app/controllers/v0/letters_generator_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'lighthouse/letters_generator/service'
+require 'lighthouse/letters_generator/service_error'
+
+module V0
+  class LettersGeneratorController < ApplicationController
+    Raven.tags_context(team: 'benefits-claim-appeal-status', feature: 'letters-generator')
+    DOWNLOAD_PARAMS = %i[
+      id
+      format
+      militaryService
+      serviceConnectedDisabilities
+      serviceConnectedEvaluation
+      nonServiceConnectedPension
+      monthlyAward
+      unemployable
+      specialMonthlyCompensation
+      adaptedHousing
+      chapter35Eligibility
+      deathResultOfDisability
+      survivorsAward
+    ].freeze
+
+    def index
+      response = service.get_eligible_letter_types(@current_user.icn)
+      render json: response
+    rescue Lighthouse::LettersGenerator::ServiceError => e
+      render json: { title: e.title, message: e.message, status: e.status }.to_json
+    end
+
+    def download
+      permitted_params = params.permit(DOWNLOAD_PARAMS)
+      letter_options = permitted_params.to_h.select { |_, v| v == 'true' }
```

Comment:
This goes with the comment on the changes in the service code. I'd like the service's code to be boolean based. Does this fit the bill in your eyes?

```ruby
letter_options = h.reduce({}) do |acc, p|
  acc[p[0]] = ActiveModel::Type::Boolean.new.cast(p[1])
  acc
end
```

Reply:
Maybe this is more idiomatic, I dunno:

```ruby
h.reduce({}) do |acc, (k, v)|
  bool_val = ActiveModel::Type::Boolean.new.cast(v)
  acc.merge(k => bool_val)
end
```

Reply:
@kjduensing Do you feel the casting should be done in the controller or the service?

Reply:
I think it should be done at the controller level. The service is more likely to be used by others than the controller is, and it seems like an arbitrary decision for the service to only except true/false in string form. What do you think?